app/services/benefit_check_service.rb

```diff
@@ -0,0 +1,45 @@
+class BenefitCheckService
+  BENEFIT_CHECKER_NAMESPACE = 'https://lsc.gov.uk/benefitchecker/service/1.0/API_1.0_Check'.freeze
+
+  attr_reader :application
+
+  def initialize(application)
+    @application = application
+  end
+
+  def check_benefits
+    soap_client.call(:check, message: benefit_checker_params).body.dig(:benefit_checker_response)
+  end
+
+  private
+
+  def benefit_checker_params
+    {
+      clientReference: application.id,
+      nino: applicant.national_insurance_number,
+      surname: applicant.last_name.strip.upcase,
+      dateOfBirth: applicant.date_of_birth.strftime('%Y%m%d'),
+      dateOfAward: Date.today.strftime('%Y%m%d')
+    }.merge(credential_params)
+  end
+
+  def credential_params
+    {
+      lscServiceName: Rails.configuration.x.benefit_check.service_name,
```

Comment:
**[not a blocker]** 💄 DRY, Given all the configurations have the same base, consider just defining a helper method:

```ruby
  def initialize(application)
   @application = application
   @config = Rails.configuration.x.benefit_check
  end

  ...

  private

  attr_reader :config

  def credential_params
    {
       lscServiceName: config.service_name,
       clientOrgId: config.client_org_id,
       clientUserId: config.client_user_id
    } 
  end 
```